packages/common/src/types/auth.ts

```diff
@@ -6,3 +10,19 @@ export enum AuthTokenPrefix {
     SERVICE_ACCOUNT = 'ldsvc_',
     PERSONAL_ACCESS_TOKEN = 'ldpat_',
 }
+
+/**
+ * The account object is used to store the account information for the user.
+ * It's meant to be agnostic of the authentication method.
+ *
+ * AuthData defaults to EmbedJwt for now as we learn more.
+ */
+export type Account<AuthData extends EmbedJwt | Record<string, unknown>> = {
```

Comment:
would it be simpler to use if we had multiple fields instead of type templating?

Reply:
I suppose it could be. I was trying to avoid encoding JWT specific concerns in the type and use generics to give flexibility.